From b2ee22409705ee043d79550867e9002ab6e86672 Mon Sep 17 00:00:00 2001
From: Liu Yiding <liuyd.fnst@fujitsu.com>
Date: Sat, 28 Feb 2026 17:01:58 +0800
Subject: [PATCH] Avoid do_configure requires check error.

The gmpy2-2.3.0 need setuptools < 80, but version in oe-core is 80.9.0,
so fix it to avoid following error.

| ERROR Missing dependencies:
|       setuptools<80,>=77

Upstream-Status: Inappropriate

Signed-off-by: Liu Yiding <liuyd.fnst@fujitsu.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 214d656..30d3f33 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ['setuptools>=77,<80', 'setuptools_scm[toml]>=6.0']
+requires = ['setuptools>=77', 'setuptools_scm[toml]>=6.0']
 build-backend = 'setuptools.build_meta'
 
 [project]
